### ISSUE TITLE
More of an idea or question

### ISSUE THREAD

**Issue by pandatoast** [aaronpkelly/splitList#1]
OK lets say the deadline is 15:00 and each task should take like 10 mins
If I now add another task, it adds it fine but I kinda expected that the deadline would be extended for another 10 mins to 15:10 or am I thinking about this wrong?

**Comment by aaronpkelly** [comment]
Yeah this is definitely a good question! So I had in mind the idea that there are:
- hard "deadlines" that cannot be breached, so no matter how many tasks you add, it will not adjust the deadline
- soft deadlines that are flexible and move when a task is added

currently it's possible to click the "set deadline and go!" button in order to reset your deadline to a future time, but you would need to decide how much additional time to give all tasks.

I'll try a few implementations out and see how they go - thank you @pandatoast!
